lib/socket.js

```diff
@@ -134,7 +134,9 @@ module.exports = class UDXSocket extends events.EventEmitter {
         host = '::'
         family = 6
         this._port = binding.udx_napi_socket_bind(this._handle, port, host, family)
-      } catch {
+      } catch (error) {
+        if (error.code === 'EADDRINUSE') throw error
```

Comment:
Maybe this is not the proper fix, although is good to throw before if we already know the port is used, but still the bind should not throw `EINVAL`

Reply:
I think the first `udx_napi_socket_bind` touches/modifies `_handle`, then on the second `udx_napi_socket_bind` call the bug happens. So even if the current fix is ok, it would be cool to understand why the second call throws `EINVAL`?